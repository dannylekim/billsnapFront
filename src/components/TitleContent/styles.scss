.title {
  width: 500px;
}

div.title__row {
  display: flex;
  flex-wrap: wrap;
  text-align: left;
}

h3.title__title {
  font-weight: bold;
  font-size: 45px;
  line-height: 71px;
}

div.title__text {
  font-size: 16px;
  font-weight: bolder;
}

button.title__button.btn.btn-primary {
  background: #3bd0a3;
  border: 1px solid #3bd0a3;
  box-shadow: 0 2px 0 #17c671;
  font-size: 15px;
  font-weight: bold;
}

button.title__button.btn.btn-primary:hover {
  background: #38b58f;
  border: 1px solid #38b58f;
}
