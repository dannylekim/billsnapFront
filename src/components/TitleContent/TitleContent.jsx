import React from "react";
import "./styles.scss";
import {Button} from "shards-react";

export const TitleContent = () => {
    return (
        <div className="title">
            <div className="title__row">
                <h3 className="title__title">Welcome to Billsnap</h3>
            </div>
            <div className="title__row title__text">
                <p>
                    BillSnap is all about snapping away full bills into dust. The goal is to facilitate the process of
                    splitting bills with roommates, friends & family.
                </p>
            </div>
            <div className="title__row">
                <Button size="md" pill className="title__button">How it works</Button>
            </div>
        </div>
    );
};

export default TitleContent;
