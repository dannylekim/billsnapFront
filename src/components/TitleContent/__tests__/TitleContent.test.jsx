import React from "react";
import ReactDOM from "react-dom";
import {TitleContent} from "../TitleContent.jsx";

describe("TitleContent", () => {
  describe("render", () => {
    it("renders without crashing", () => {
      const div = document.createElement("div");
      ReactDOM.render(<TitleContent />, div);
    });
  });
});

describe("TitleContent", () => {
  describe("render", () => {
    describe("snapshots ðŸ“¸", () => {
      it("TitleContent should match snap shot", () => {
        matches(<TitleContent />);
      });
    });

    describe("components", () => {});
  });

  describe("functions", () => {});
});
