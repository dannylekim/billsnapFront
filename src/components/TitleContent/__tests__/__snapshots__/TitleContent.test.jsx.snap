// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleContent render snapshots ðŸ“¸ TitleContent should match snap shot 1`] = `
<div
  className="title"
>
  <div
    className="title__row"
  >
    <h3
      className="title__title"
    >
      Welcome to Billsnap
    </h3>
  </div>
  <div
    className="title__row title__text"
  >
    <p>
      BillSnap is all about snapping away full bills into dust. The goal is to facilitate the process of splitting bills with roommates, friends & family.
    </p>
  </div>
  <div
    className="title__row"
  >
    <button
      className="title__button btn btn-primary btn-md btn-pill"
      onClick={[Function]}
    >
      How it works
    </button>
  </div>
</div>
`;
